index.html

```diff
@@ -9,12 +9,134 @@
     <link rel="shortcut icon" href="images/lightbeam-48.png" type="image/x-icon">
     <link rel="icon" href="images/lightbeam-48.png" type="image/x-icon">
     <link rel="stylesheet" href="css/style.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
     <script src="js/store.js" type="text/javascript"></script>
     <script src="js/viz.js" type="text/javascript"></script>
   </head>
   <body>
-    <h1>Lightbeam</h1>
-    <canvas id="canvas" width="300" height="300"></canvas>
+    <aside class="controls">
+    <header>
+      <img class="logo" role="banner" src="images/lightbeam_150x45.png
+">
+    </header>
+    <nav role="navigation">
+      <h2>Visualization</h2>
+      <div class="nav__vis">
+        <button class="nav__button nav__button--vis"><img class="icon icon--button" src="images/lightbeam_icon_graph.png">Graph</button>
+        <button class="nav__button nav__button--vis"><img class="icon icon--button" src="images/lightbeam_icon_list.png">List</button>
+      </div>
+      <h2>Data</h2>
+      <div class="nav__data">
+        <button class="nav__button nav__button--data"><img class="icon icon--button" src="images/lightbeam_icon_download.png">Save Data</button>
+        <button class="nav__button nav__button--data"><img class="icon icon--button" src="images/lightbeam_icon_reset.png">Reset Data</button>
+      </div>
+      <div class="nav__links">
+        <img class="icon" src="images/lightbeam_icon_feedback.png">
+        <a href="mailto:lightbeam-feedback@mozilla.org">Give Us Feedback</a>
+        <br>
+        <a href="https://support.mozilla.org/en-US/kb/disable-or-remove-add-ons">Uninstall Lightbeam</a>
+      </div>
+    </nav>
+    </aside>
+    <main role="main">
+      <section class="top-bar">
+        <div class="top-bar__summary">
+          <span>
+            <h2>Data Gathered Since</h2>
+            <h3 id="date-gathered">June 13, 2017</h3>

```

Comment:
Can you add TODO HTML comments to anything hard coded? It's easy to miss things later on and forget it's not actually working.